### PR TITLE
refactor(workers): Remove depNameSanitized

### PR DESCRIPTION
**Pull request by zharinov**
## Context:

Ref: #8075
Ref: #9114

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/workers/repository/updates/flatten.ts [hunk 1]
```diff
@@ -17,16 +17,6 @@ const upper = (str: string): string =>
 export function applyUpdateConfig(input: BranchUpgradeConfig): any {
   const updateConfig = { ...input };
   delete updateConfig.packageRules;
-  // TODO: Remove next line once #8075 is complete
```
Comment (zharinov) [inline lib/workers/repository/updates/flatten.ts:20]: #8075

**Review by viceice — COMMENTED** [review]
`depNameSanitized` isn't this used in render templates?

**Comment by zharinov** [comment]
Well, I see "remove next line" wasn't said literally...

**Comment by rarkins** [comment]
I don't think this will work while we still have lots of references to `depNameSanitized` in templates. Maybe our TODO got things mixed up

**Comment by rarkins** [comment]
We maybe got it mixed up with depNameShort

**Pull request closed without merging**